pkg/apis/config/v1beta1/config.go

```diff
@@ -81,18 +80,12 @@ type ElasticSearch struct {
 	Password string `json:"password,omitempty"`
 }
 
-// ReservedExcessCapacity holds information about additionally deployed reserved excess capacity pods.
-type ReservedExcessCapacity struct {
-	// Replicas is the amount of reserve excess capacity pods.
-	Replicas int32 `json:"replicas"`
-
-	// Resources specifies the resources of the single excess capacity pods
-	// + optional
-	Resources *corev1.ResourceRequirements `json:"resources,omitempty"`
-}
-
-// Ingress holds information about a ingress
-type Ingress struct {
-	Enabled bool   `json:"enabled"`
-	Host    string `json:"host"`
+// HealthCheckTarget specifies a deployment whose health should be checked.
+type HealthCheckTarget struct {
+	//Namespace specifies the namespace where resources relevant for a health check exist in.
+	Namespace string `json:"namespace,omitempty"`
+	// DeploymentName is the name of a deployment whose health will be checked.
+	DeploymentName string `json:"deploymentName,omitempty"`
+	//IntervalSeconds specifies the frequency of the health check in seconds
+	Interval metav1.Duration `json:"interval,omitempty"`
```

Comment:
we should default this value or do not run any healthcheck if `Interval == 0`.
otherwise we would ddos the apiserver with health requests.

nit: the docstring has to be adjusted `/IntervalSeconds/Interval`